erddap/content/datasets.xml

```diff
@@ -2176,8 +2240,10 @@
       </sourceAttributes -->
       <addAttributes>
         <att name="_FillValue" type="float">NaN</att>
-        <att name="ioos_category">Unknown</att>
+        <att name="ioos_category">Temperature</att>
         <att name="long_name">S27 CIL</att>
+        <att name="standard_name">sea_water_temperature</att>
+        <att name="units">1</att>
```

Comment:
The unit here should, likely, be degree_C

Reply:
Same comment as first comment applies, happy to make this change, either way.